lib/hooks/useNetwork.js

```diff
@@ -9,11 +9,20 @@ import { WalletContext } from 'lib/components/WalletContextProvider'
 export const networkAtom = atom({})
 
 export const useNetwork = () => {
-  const [network, setNetwork] = useAtom(networkAtom)
-  const [errorState, setErrorState] = useAtom(errorStateAtom)
   const router = useRouter()
-  const networkName = router.query.networkName
   const walletContext = useContext(WalletContext)
+  
+  const [network, setNetwork] = useAtom(networkAtom)
+  const [errorState, setErrorState] = useAtom(errorStateAtom)
+  
+  const poolAlias = router.query.poolAlias
```

Comment:
Should we hardcode the `chainId` along with the alias addresses? Then we can just read from there and not worry about this. That'll make adding an alias for rinkeby easier in the future.

Reply:
So have something like:
```
POOL_ALIASES = {
  'basis': {
    chainId: 1,
    poolAddress: '0xface'
  }
}
```

?

Reply:
Yeah that's what I was thinking